### ISSUE TITLE
"Is there only one training model? some questions regarding the training code."

### ISSUE THREAD

**Issue by JahaoZhang** [wukui-muc/Offline_RL_Active_Tracking#5]
Thank you very much for your open-source code. I would like to ask: the project includes a pre-trained model, which seems to be used in the FlexibleRoom environment. Is the same model also used when evaluating in other environments?
While reading the code, I noticed that the train_offline.py file seems to be a training script used for ablation experiments and does not include logic related to segmentation mask processing. Therefore, I would like to ask if you could provide the code that contains the complete training pipeline, especially the part involving segmentation mask handling. Alternatively, if such a script already exists in the repository, could you kindly point me to its specific location?

**Comment by JahaoZhang** [comment]
Thank you very much for your response. I am currently trying to reproduce the results reported in the paper, and I would like to know the exact parameter values used when running the model in each environment—specifically --env, --chunk_size, --min_mid_term_frames, --max_mid_term_frames, --detection_every, and --prompt. I am especially interested in the settings for the SnowForest scene. I would greatly appreciate any information you can provide!

<img width="1445" height="383" alt="Image" src="https://github.com/user-attachments/assets/b8a9f514-ce3c-4478-a35f-8cfaf6a07c7f" />

**Comment by JahaoZhang** [comment]
Thank you for your answer, but when I plan to evaluate the snowforest environment, I don’t know what value I should set for the --env parameter. There is no JSON file for snowforest in the general folder. Could you tell me the parameters for the snowforest environment, especially the --env parameter?

**Comment by wukui-muc** [comment]
UnrealTrackGeneral-SnowForest-ContinuousColor-v0
or UnrealTrackMulti-SnowForestNav-ContinuousColor-v0

**Comment by JahaoZhang** [comment]
Thank you very much for your answer. I wish you all the best.

**Comment by JahaoZhang** [comment]
Thank you for your answer, but when I plan to evaluate the urbanroad environment, I don’t know what value I should set for the --env parameter. Because when I set the parameter to UnrealTrackGeneral-UrbanRoad-ContinuousColor-v0, the results differ quite a lot from those in the paper, I’d like to ask what your exact parameters were.
